mobile_verifier/src/heartbeats.rs

```diff
@@ -231,10 +252,11 @@ impl Heartbeat {
         Ok(
             sqlx::query_as::<_, HeartbeatSaveResult>(
                 r#"
-                INSERT INTO heartbeats (cbsd_id, hotspot_key, cell_type, latest_timestamp, truncated_timestamp)
-                VALUES ($1, $2, $3, $4, $5)
+                INSERT INTO heartbeats (cbsd_id, hotspot_key, cell_type, latest_timestamp, truncated_timestamp, coverage_object)
```

Comment:
did i miss a migration to add the coverage_object uuid to the heartbeats table?

Reply:
yup!

Reply:
I mean - nope! I forgot to add it! my b lol